src/settings.yml

```diff
@@ -0,0 +1,44 @@
+networks:
+  sepolia:
+    rpc: https://rpc.ankr.com/eth_sepolia
+    chain-id: 11155111
+    currency: ETH
+  base:
+    rpc: https://base.llamarpc.com
+    chain-id: 8453
+    currency: ETH
+  flare:
+    rpc: https://rpc.ankr.com/flare	
+    chain-id: 14
+    currency: FLR
+
+subgraphs:
```

Comment:
pls remove subgraphs as we aren't testing them yet

Reply:
Done